iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -99,7 +99,7 @@ function localName(e) {
 }
 
 Viewer.prototype._addDocumentSetTabs = function() {
-    if (this._report.isDocumentSet()) {
+    if (this._report.isDocumentSet() && this._report.documentSetFiles().length > 1) {
```

Comment:
Why is this change required?  It looks like docSetFiles is the same for stub vs default.

Reply:
In stub mode, docSetFiles is an array containing the iXBRL files, even if there's only one.  In normal mode, docSetFiles is undefined.

I've now moved the length check inside `isDocumentSet()`, and refactored the other code that used this method (30ae2ec) so `isDocumentSet` is back to its original meaning of "is a multi-document document set".